pkgs/development/tools/rust/racerd/default.nix

```diff
@@ -2,6 +2,8 @@
 
 with rustPlatform;
 
+with import <nixpkgs> {};
```

Comment:
importing  `<nixpkgs>` is not allowed in nix expressions, you need to pass `Security` as an argument. e.g. https://github.com/NixOS/nixpkgs/commit/8166fc21b87

Reply:
thank you, I really appreciate you pointing me in the right direction